src/Search/SortableInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Search;
+
+interface SortableInterface
```

Comment:
While you can find many `*Interface` (& `Abstract*`) classes, we try to avoid that with new additions.
`interface Sortable` should be clear enough: see https://blog.nette.org/en/prefixes-and-suffixes-do-not-belong-in-interface-names

Reply:
Done